nfl/react-wildcat
# Upgraded webpack to version 4

To test run 
> node ./shell/install.js
> node ./shell/install-example.js
and then go to `example` dir and run
> yarn dev
If there are errors make sure there is webpack version 4 in `node_modules` of `react-wildcat`
